spec/requests/shopping_lists_spec.rb

```diff
@@ -10,7 +10,7 @@
     }
   end
 
-  describe 'POST /shopping_lists' do
+  describe 'POST games/:game_id/shopping_lists' do
```

Comment:
The titles of these example groups were not updated when the Game model was introduced.